README.md

```diff
@@ -181,10 +181,34 @@ ShopifyCheckoutSheetKit.preloading.enabled = false
 ShopifyCheckoutSheetKit.preload(checkout: checkoutURL) // does nothing as preloading.enabled: false while preloading=false 
 ```
 
+### Preload Invalidation
+
+_N.B if you call `preload` anywhere in your app code, preload invalidation becomes very important to understand.`_
+
+Preloading is powered by a cache. To keep the cached view in sync with changes (eg to cart) we need to invalidate from time to time. This is a shared responsibility between the internal SDK and the merchant developer. The internal SDK will do it when it can but there are occasions that the merchant developer will need to invalidate. There are currently two ways to invalidate:
+
+* call `preload` (this will invalidate any existing cache and re-cache with current checkout state)
+* Set `ShopifyCheckoutSheetKit.preloading.enabled` to false (entirely disables preloading, including cache)
+
+The SDK invalidates preloading (with no refresh) when:
+* The server returns a non 2XX response code
+* There is a network error
+* A checkout is completed (indicated by server response)
+* The `ShopifyCheckoutSheetKit.Configuration` object is manipulated (e.g theming changes)
+* The checkout sheet is closed _and_ the merchant developer has _not_ called the `preload` function
+
+The Merchant developer needs to invalidate and refresh the cache _only after_ they have called `preload` at least once and:
```

Comment:
I wasn't 100% either tbh. Any suggestions?

Reply:
Since you're speaking directly to the person reading I'm think it would make more sense to use the term "you" more.

```
"It is important that you invalidate and refresh the cache _only after_ you have called `preload` at least once."
```